README.md

```diff
@@ -46,35 +57,54 @@ The CTO, first engineer, cofounder, or anyone at a pre-seed startup who is accou
   - [ ] Define and document process and expectations for code reviews
     - https://mtlynch.io/human-code-reviews-1/
     - https://mtlynch.io/human-code-reviews-2/
-  
+  - [ ] Early code will likely hit product limitations before performance/scalability becomes an issue; focus accordingly
+  - [ ] Seed development env databases with code
+
 - [ ] Engineering Team
+  - [ ] Hanlon's razor: Never attribute to malice that which is adequately explained by ignorance/stupidity/lack of knowledge.
   - [ ] Always be looking forward and "recruiting"
   - [ ] Once you have at least one more engineer, define engineering levels 
   - [ ] Set up a structure to allow for experimentation
   - [ ] QA: You will need QA earlier than you think you do
   - [ ] When you are no longer effective managing the team, look for a Manager/Director/VP of Engineering who is 10x better than you
+    - Choose to manage or write code. Doing both will result in poor performance of both.
 
 - [ ] Day-to-day execution
   - [ ] stand-ups, sprints, retros, milestones
   - [ ] Use simple "project management" to keep a healthy product development rhythm, no matter how small your "team"
+  - [ ] A team member's failure likely stems from a decision you made (or didn't make)
 
 - [ ] Product
   - [ ] Use "feature flags" to release new features and selectively enable them for existing customers.
   - [ ] Don't make the thing more awesome, always look for how to make your users more awesome
   - [ ] To make a decision, look for answers in analytics and metrics
   - [ ] Reporting: Sooner or later, the data you can query from the database will need to be turned into reporting features for your customers
-
+  - [ ] Customer suggested solutions are usually wrong. Dig into the "why" before weighing the "how"; the need is usually legitimate
+  - [ ] Never implement a feature just for one customer; it will hurt you more in the future than help you in the short term
+    - Exception of paid/sponsored features, which should take maintenance and support into consideration
+  - [ ] Ensure existing customers want a feature before you start building it
+    - Potential customers always want the moon; rarely will just-one-more-feature! start landing customers. Choose features many leads want, not just a few
+  - [ ] Have a vision for your product and work towards it. Passing on customers because of features that don't fit your vision will be painful, but will pay off in the long run.
+  - [ ] Users are stupid. Dumb down features/UIs until a flea could use them.
```

Comment:
@danielwestendorf , can we update this to less strong language?
Suggestion:
- [ ] Aim for a "simple" product. Simple is not easy, but it's what your customers and users need. The craft of product is doing the hard work of figuring out the complexity to make using it simple for your customers.

Reply:
```suggestion
  - [ ] Aim for a "simple" product. Simple is not easy, but it's what your customers and users need. The craft of product is doing the hard work of figuring out the complexity to make using it simple for your customers.
```